scIB/metrics.py

```diff
@@ -4,6 +4,7 @@
 import pandas as pd
 import matplotlib.pyplot as plt
 import seaborn as sns
+import sklearn
```

Comment:
do you need this? Or does sklearn.metrics suffice?

Reply:
We also want to use `sklearn.linear_model` for the PCR score, so I'd keep it the way it is. Also, it's nicer to see `sklearn.metrics` rather than `metrics` so we know exactly where that function comes from. I removed all imports from within functions, as they would only cause hidden errors

Reply:
okay

Reply:
this doesn't work. sklearn doesn't automatically import subpackages. I think you need an `import sklearn.metrics` there as well